inc/woocommerce.php

```diff
@@ -16,7 +16,18 @@
  * @return void
  */
 function _s_woocommerce_setup() {
-	add_theme_support( 'woocommerce' );
+	add_theme_support( 'woocommerce', array(
+		'thumbnail_image_width' => 150,
+		'single_image_width'    => 300,
+		'product_grid'          => array(
+			'default_rows'    => 3,
+			'min_rows'        => 2,
+			'max_rows'        => 8,
```

Comment:
```suggestion
			'max_rows'        => '',
```
Same here. We should not restrict the maximum rows. This is the WooCommerce default.